noodles-editor/public/noodles/california-earthquakes/README.md

```diff
@@ -0,0 +1,40 @@
+# California Earthquakes
```

Comment:
It'd be good to give credit to where each of these were adapted from, like Kepler, etc